app/views/goals/show.html.erb

```diff
@@ -51,7 +51,7 @@
               <td><%= link_to habit.name, edit_goal_habit_path(@goal, habit), style: "text-decoration: none" %></td>
               <td><%= habit.description.truncate(15) %></td>
               <td>
-                <%= link_to "#{habit.completed_today? ? 'completed' : 'in process'}",
+                <%= link_to "#{habit.completed_today? ? 'uncomplete' : 'complete'}",
```

Comment:
I would move this to the helper method named `habit_link_name` or something like that.